config/testgrids/config.yaml

```diff
@@ -8,6 +8,7 @@ dashboard_groups:
     - gardener-dependency-watchdog
     - gardener-extension-networking-cilium
     - gardener-extension-networking-calico
+    - gardener-landscaper
```

Comment:
If you don't use the dashboard, you have to revert the change. Otherwise, the testgrid test fails.